cmake/modules/FindPythonLibsNew.cmake

```
@@ -52,9 +52,9 @@ endif()
 
 # Use the Python interpreter to find the libs.
 if(PythonLibsNew_FIND_REQUIRED)
-  find_package(PythonInterp ${PythonLibsNew_FIND_VERSION} REQUIRED)
+  find_package(Python ${PythonLibsNew_FIND_VERSION} REQUIRED EXACT COMPONENTS Interpreter Development)
```

Use `Development.Module` to be compatible with static libraries as well. Important for wheels :)

```suggestion
  find_package(Python ${PythonLibsNew_FIND_VERSION} REQUIRED EXACT COMPONENTS Interpreter Development.Module)
```